### ISSUE TITLE
Partials Not Rendering With Mustache Templates

### ISSUE THREAD

**Issue by harshadyeola** [datafolklabs/cement#320]
I find that cement does not find the partial path with  cement extension mustache.
I tried to use partial with mustache extension but it did not find partial path.
 I guess cement did not propagate the path of partial.

Any suggestion over using partials?

**Comment by derks** [comment]
I'm not following what you mean by `partial path` exactly.  Please clarify, and provide a code example if possible.

**Comment by harshadyeola** [comment]
Here is My code snippet, that explains the thing

```
myfile = open('/etc/my.cnf', encoding='utf-8',
                          mode='w')

self.app.render((data), 'virtualconf.mustache',
                out=myfile)
myfile.close()
```

```
virtualconf.mustache

Hello i am in virtualconf.mustache
{{> php}}

php.mustache

"Hello I am using partials"
```


With above code i get output

```
$cat /etc/my.cnf
Hello i am in virtualconf.mustache
```

**Comment by derks** [comment]
Confirmed, I can reproduce this locally.

 * Ref: https://mustache.github.io/mustache.5.html

**Comment by derks** [comment]
Fix pushed to `master`, will be released with Cement 2.8.0 stable, and Cement 2.6.2 Stable.

**Comment by harshadyeola** [comment]
thanks @derks for this quick fix
looking forward for the release of this fix

**Comment by derks** [comment]
It was released earlier today in 2.6.2 (current stable).

**Comment by harshadyeola** [comment]
Ahh! awesome thank you.
